bundles/org.openhab.binding.venstarthermostat/src/main/java/org/openhab/binding/venstarthermostat/internal/dto/VenstarInfoData.java

```diff
@@ -25,20 +25,29 @@
     VenstarSystemState state;
     VenstarSystemMode mode;
     VenstarAwayMode away;
+    VenstarFanMode fan;
+    VenstarFanState fanstate;
+    VenstarScheduleMode schedule;
+    VenstarSchedulePart schedulepart;
```

Comment:
private should be added I guess.
Please also fix the other members.

Reply:
Yes. OK.